### PR TITLE
[ADD] web_widget_iframe: new module to display an iframe in a form view

### PR DESCRIPTION
**Pull request by benwillig**
This modules simply adds an form widget which display an iframe.

### CODE REVIEW

File: web_widget_iframe/README.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/web-10-0/web-10-0-web_widget_iframe
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/162/10.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module simply adds a new widget which display an Iframe for a given
```
Comment (tarteo) [inline web_widget_iframe/README.rst:28]: Can you add an usage section

File: web_widget_iframe/static/src/js/web_widget_iframe.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+odoo.define('web_widget_iframe.web_widget_iframe', function(require) {
```
Comment (tarteo) [inline web_widget_iframe/static/src/js/web_widget_iframe.js]: Can you rename the namespace to web_widget_iframe.FieldUrlIframe

File: web_widget_iframe/static/src/js/web_widget_iframe.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+    var core = require('web.core');
+
+    var FieldChar = core.form_widget_registry.get('char');
+
+    var FieldUrlIframe = FieldChar.extend({
+        template: 'FieldIframe',
+
+        render_value: function() {
+            var self = this;
```
Comment (tarteo) [inline web_widget_iframe/static/src/js/web_widget_iframe.js:12]: There's not really a reason for this variable you can just use `this`.

File: web_widget_iframe/static/src/js/web_widget_iframe.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+odoo.define('web_widget_iframe.web_widget_iframe', function(require) {
+    "use strict";
+
+    var core = require('web.core');
+
+    var FieldChar = core.form_widget_registry.get('char');
```
Comment (tarteo) [inline web_widget_iframe/static/src/js/web_widget_iframe.js:6]: You can also use FieldUrl which maybe automatically adds the protocol etc (don't know for sure :P).

**Review by tarteo — CHANGES_REQUESTED** [review]
Very nice! Can you also look at the lint errors?

**Comment by pedrobaeza** [comment]
Isn't this a bit of security whole allowing any external object to catch data from backend?

**Comment by hbrunn** [comment]
you should set `sandbox=""`, cf https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe to have this locked down by default and possibly add an option to the widget to selectively add permissions

**Comment by benwillig** [comment]
@hbrunn Thanks for your feedback. I added the empty sandbox attribute by default and it can be overriden by specifying it on the field tag.

@tarteo Fixed the code based on your notes.

@pedrobaeza What do you mean?

**Comment by pedrobaeza** [comment]
@benwillig the sandbox mode suggested by @hbrunn answers my concern.

File: web_widget_iframe/README.rst (hunk 1 above)
Comment (pedrobaeza) [reply web_widget_iframe/README.rst:28]: Yes, please, with the `sandbox` part included.

**Review by tarteo — APPROVED** [review]

**Pull request closed without merging**